components/navbar/profileDropdown.tsx

```diff
@@ -61,19 +64,21 @@ export default function ProfileDropdown({ title, onClose }: Props) {
           </OrganizationProfile>
         </Frame1>
         <Frame2>
-          <ProfileItemDiv
-            onClick={() => {
-              router.push('/setting');
-            }}
-          >
-            <SVGIcon
-              name='setting-icon'
-              width='12px'
-              height='12px'
-              viewBox='0 0 12 12'
-            />
-            <p>Settings</p>
-          </ProfileItemDiv>
+          {currentWorkspace?.role !== 'MEMBER' && (
```

Comment:
We will check here (currentWorkspace?.role !== UserRole.ADMIN || currentWorkspace?.role !== UserRole.OWNER) && ...

Reply:
Done